HacktoberfestProject.Web/Controllers/HomeController.cs

```diff
@@ -48,13 +49,27 @@ public async Task<IActionResult> Index()
         [HttpGet]
         public IActionResult Add()
         {
-            //How to access claims -left here for reference
-            ViewBag.Username = _contextAccessor.HttpContext.User
-                .Claims.FirstOrDefault(c => c.Type == GitHubUsernameClaimType).Value;
-
             return View();
         }
 
+        [Authorize]
+        [HttpPost]
+        public async Task<IActionResult> Add(AddPrViewModel vm)
+        {
+            if (!ModelState.IsValid)
+            {
+                return View("Add", vm);
+            }
+
+            vm.UserName = _contextAccessor.HttpContext.User
+                .Claims.FirstOrDefault(c => c.Type == GitHubUsernameClaimType)?.Value;
+
+            await _tableService.AddPrAsync(vm.UserName, vm.Owner, 

```

Comment:
TableService returns a response - `ServiceResponse<PullRequest>` perhaps we could check the status as this will tell us if the record was added successfully or something else happened.

Reply:
What would you like that to do throw an exception, direct to an error page?

Reply:
see above